pandas/tests/arrays/test_timedeltas.py

```diff
@@ -303,18 +313,20 @@ def test_std(self):
         assert isinstance(result, pd.Timedelta)
         assert result == expected
 
-        result = nanops.nanstd(np.asarray(arr), skipna=True)
-        assert isinstance(result, pd.Timedelta)
-        assert result == expected
+        if getattr(arr, "tz", None) is None:
+            result = nanops.nanstd(np.asarray(arr), skipna=True)
+            assert isinstance(result, pd.Timedelta)
+            assert result == expected
 
         result = arr.std(skipna=False)
         assert result is pd.NaT
 
         result = tdi.std(skipna=False)
         assert result is pd.NaT
 
-        result = nanops.nanstd(np.asarray(arr), skipna=False)
-        assert result is pd.NaT
+        if getattr(arr, "tz", None) is None:
+            result = nanops.nanstd(np.asarray(arr), skipna=False)
```

Comment:
are we sure the actual result is correct? e.g. you are just comparing vs the implementation itself. IOW a result which checks a fixed series of datetimes for mean/std would be good.

Reply:
agreed these tests needs fleshing out.  im still giving thought to where they should live

Reply:
sure, that's fine about location. i am more concerned with if the results are actually correct; reasoning is that they are converted to i8 but need to make sure that these are coming back correct as well. i am not sure this is actually tested.

Reply:
Yes, we have tests with explicit `expected` for std() with td64 in test_reductions, test_describe, and test_analytics.